### PR TITLE
Carplay events modified to adhere to server schema.

### PR DESCRIPTION
**Pull request by vincethecoder**
- Updated the carplay event constants to match up with the expected events schema at the server side.

### CODE REVIEW

**Review by rclee — APPROVED** [review]
👍

File: MapboxMobileEvents/MMENamespacedDependencies.h [hunk 1]
```diff
@@ -425,6 +425,10 @@
 #define MMEEventTypeLocalDebug __NS_SYMBOL(MMEEventTypeLocalDebug)
 #endif
 
+#ifndef MMEEventTypeOfflineDownload
```
Comment (akitchen) [inline MapboxMobileEvents/MMENamespacedDependencies.h:428]: What's this about? Is this related to CarPlay events at all?

File: MapboxMobileEventsTests/MMEEventsManagerTests.mm [hunk 2]
```diff
@@ -829,20 +829,6 @@ @interface MMEEventsManager (Tests) <MMELocationManagerDelegate>
                 });
             });
             
-            context(@"when a carplay event is pushed", ^{
```
Comment (akitchen) [inline MapboxMobileEventsTests/MMEEventsManagerTests.mm:832]: Why delete this test? It seems it should be updated for the new event name?

File: MapboxMobileEvents/MMENamespacedDependencies.h (hunk 1 above)
Comment (rclee) [reply MapboxMobileEvents/MMENamespacedDependencies.h:428]: No, this is a separate event related to Maps SDK

File: MapboxMobileEventsTests/MMEEventsManagerTests.mm (hunk 2 above)
Comment (rclee) [reply MapboxMobileEventsTests/MMEEventsManagerTests.mm:832]: It's no longer needed since the prefix check for `navigation` already exists
